packages/core/src/rpc/substrate/chain.ts

```diff
@@ -71,6 +71,20 @@ export const chain_getBlock: Handler<
   }
 }
 
+/**
+ * @param context
+ * @param params - [`blockhash`]
+ *
+ * @return Block extrinsics
+ */
+export const archive_unstable_body: Handler<[HexString], HexString[]> = async (context, [hash]) => {
```

Comment:
Would you like me to create a new `archive` folder, and put this function there?

Reply:
Up to you

Reply:
it will make finding things a bit easier

Reply:
done, I also moved the archive alias.